### PR TITLE
add goimports package

### PR DESCRIPTION
**Pull request by sofuture**
Add a package for [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports).

### CODE REVIEW

File: pkgs/development/tools/goimports/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, go, fetchurl, fetchgit, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  rev = "7534f4943d94a318edde90212439e538ed54cdde";
+  version = "${lib.strings.substring 0 7 rev}";
```
Comment (jagajaga) [inline pkgs/development/tools/goimports/default.nix]: Change `version` to a date of creation of that revision (in `YYYY-MM-DD` format).
See https://github.com/NixOS/nixpkgs/pull/6539

**Comment by jzellner** [comment]
@jagajaga :+1:

**Comment by jagajaga** [comment]
Thanks. But squash commits please.

**Comment by jzellner** [comment]
@jagajaga done, thanks

**Comment by jagajaga** [comment]
Oh, I'm sorry, but can't this pkg be built with `buildGoPackage`? https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/go-packages.nix

**Comment by jzellner** [comment]
There's a reason I didn't use `buildGoPackage`... but that reason is probably 'I am dumb and still don't grok nix syntax'. I don't really mind taking another stab at it though, I'll close this PR for now.

**Comment by jagajaga** [comment]
I'll pack it.

**Comment by lethalman** [comment]
It's fine, thanks.

**Pull request merged**